lib/server.ts

```diff
@@ -271,6 +265,81 @@ async function main(){
         },
     );
 
+    app.post(
+        '/config',
```

Comment:
```suggestion
        '/setup',
```
Considering setup is what this endpoint does, I would recommend keeping it like that. 
Cleaner, more descriptive that way.

Reply:
Ahh reading more I can see setup is not what this endpoint does. But, still could be a good suggestion to differentiate between the 2?